codespell_lib/data/dictionary.txt

```diff
@@ -16564,7 +16574,8 @@ lpatform->platform
 lsat->last
 lsit->list
 lsits->lists
-lukid->likud
+lukid->lucid, likud,
```

Comment:
likud is the name of an israelian party. should it be a suggestion?

Reply:
Yeah probably. The fact someone added that as the initial entry hopefully means they found it as a typo. I think putting it second is right though. Should it be capitalised?